pkgs/servers/homepage-dashboard/default.nix

```
@@ -0,0 +1,76 @@
+{ buildNpmPackage
+, fetchFromGitHub
+, nodePackages
+, python3
+, lib
+, fetchpatch
+, makeWrapper
```

```suggestion
, makeBinaryWrapper
```

I think it's better to use this to avoid issue on darwin arch.